pages/dkp/kaptain/1.2.0-1.1.0/operations/minio/index.md

```diff
@@ -54,22 +49,19 @@ kubectl kudo install ./kubeflow-1.2.0_1.1.0.tgz \
       --create-namespace
 ```
 
-## Expanding MinIO cluster
+## Expanding the MinIO cluster
 
-MinIO cluster can be expanded by increasing the number of server pools in the cluster. 
+The MinIO cluster can be expanded by increasing the number of server pools in the cluster. 
 <div style="color: #8a6d3b; background-color: #fcf8e3; border-color: #faebcc; padding: 15px; margin-top: 10px; margin-bottom: 10px; border: 1px solid transparent; border-radius: 4px;">
-<p class="message--warning"><strong>WARNING: </strong>Extending an existing server pool is not supported. The cluster 
-can only be scaled linearly, by using the initial server pool as a base scale unit.</p>
+<p class="message--warning"><strong>WARNING: </strong>Expand an existing server pool is not supported. The cluster can only by adding additional server pools.</p>
```

Comment:
Sorry, seems like there was some kind of typo in my previous comment! My apologies!

```suggestion
<p class="message--warning"><strong>WARNING: </strong>Expanding an existing server pool is not supported. The cluster can only by expanded by adding additional server pools.</p>
```